### PR TITLE
Add readme file and fix UI issue

### PR DESCRIPTION
**Pull request by Krishanx92**
Add README file to store webapp static file override.

### CODE REVIEW

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/override/Readme.txt [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# Override public folder
+If you want to override a certain store webapp static files, this is the correct place to do it.
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/override/Readme.txt]: need to replace store with devportal

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Credentials/Credentials.jsx [hunk 2]
```diff
@@ -32,6 +32,7 @@ import Application from 'AppData/Application';
 import SubscribeToApi from 'AppComponents/Shared/AppsAndKeys/SubscribeToApi';
 import { ScopeValidation, resourceMethods, resourcePaths } from 'AppComponents/Shared/ScopeValidation';
 import { ApiContext } from '../ApiContext';
+import AuthManager from '../../../../data/AuthManager';
```
Comment (isharac) [inline features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Credentials/Credentials.jsx]: we can use import AuthManager from 'AppData/AuthManager';  instead?

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/override/Readme.txt (hunk 1 above)
Comment (Krishanx92) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/site/override/Readme.txt]: Fixed

File: features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Credentials/Credentials.jsx (hunk 2 above)
Comment (Krishanx92) [reply features/apimgt/org.wso2.carbon.apimgt.store.feature/src/main/resources/devportal/source/src/app/components/Apis/Details/Credentials/Credentials.jsx]: Fixed

**Pull request merged**